packaging/markers.py

```diff
@@ -62,6 +62,9 @@ def __str__(self) -> str:
     def __repr__(self) -> str:
         return f"<{self.__class__.__name__}('{self}')>"
 
+    def __eq__(self, other: Any) -> bool:
+        return bool(self.value == other.value)
```

Comment:
Why the `bool()` call?

Reply:
Currently `mypy` seems to be struggling with that. By running `nox -s lint`, I have the following error back:

```
packaging/markers.py:66: error: Returning Any from function declared to return "bool"
```

If that is fine, I can silence this mypy error with an inline `# type: ignore`. Please let me know if that solution would work for the `packaging` project (some projects don't like to silence linters)

Reply:
Nope, that's fine then.